### ISSUE TITLE
Erroneously tries to validate class variables

### ISSUE THREAD

**Issue by wadetregaskis-linkedin** [mkorpela/overrides#33]
```
from overrides import EnforceOverrides, overrides

class SuperClass(EnforceOverrides):
    someClassVariable = 'foo'

class SubClass(SuperClass):
    someClassVariable = 'bar'
```

Produces:

```
Traceback (most recent call last):
  File "test.py", line 8, in <module>
    class SubClass(SuperClass):
  File "/Users/wtregask/Documents/c2s_backend_6/build/c2s-backend/venv/lib/python3.7/site-packages/overrides/enforce.py", line 19, in __new__
    'Method %s overrides but does not have @overrides decorator' % (name)
AssertionError: Method someClassVariable overrides but does not have @overrides decorator
```

Unfortunately, there's nothing the programmer can do about that - class variables cannot be decorated.

Overrides should ignore any attributes of the class that aren't callable.

**Comment by mkorpela** [comment]
Thanks for the bug report! Fixed in https://github.com/mkorpela/overrides/commit/37a986eb8c35846a34ba5656b49d8b033adcc86a released in 2.8.0

**Comment by wadetregaskis-linkedin** [comment]
Thanks for the super fast turnaround - that's awesome!
